### ISSUE TITLE
Stop using `application/dart` script tags

### ISSUE THREAD

**Issue by natebosch** [dart-lang/stagehand#468]
We can drop dart_to_js_script_rewriter and package:browser and update the templates so that script tags always reference the compiled JS.

**Comment by natebosch** [comment]
Blocked on updating to 2.0.0 SDK and angular 5 in the templates.

**Comment by chalin** [comment]
Done via #464. Merged into the [2-dev](https://github.com/dart-lang/stagehand/tree/2-dev) branch.

cc @kwalrath @kevmoo

**Comment by natebosch** [comment]
Ah cool. Didn't realize we already had a branch going for this.
